jdaviz/configs/specviz2d/helper.py

```diff
@@ -132,6 +143,11 @@ def load_data(self, spectrum_2d=None, spectrum_1d=None, spectrum_1d_label=None,
             Show data in viewer(s).
 
         """
+        kwargs.update({
+            "spectrum_2d_viewer_reference_name": self._default_spectrum_2d_viewer_reference_name,
+            "spectrum_viewer_reference_name": self._default_spectrum_viewer_reference_name,
+            "table_viewer_reference_name": self._default_table_viewer_reference_name
+        })
```

Comment:
I've noticed this in the other helpers too: Why do we need to copy the viewer reference names into `kwargs` when we load data? I don't see the actual method using it, and if it's because we need to pass it to other methods, would it be more explicit to pass them in directly? If it's the latter, I'd feel this kind of obscures who needs it and why